packages/tables/resources/views/index.blade.php

```diff
@@ -587,7 +588,7 @@ class="fi-ta-selection-indicator"
                                 color="primary"
                                 tag="button"
                                 x-on:click="selectAllRecords"
-                                :x-show="'canSelectAllRecords(' . \Illuminate\Support\Js::from($selectsCurrentPageOnly) . ')'"
+                                x-show="canSelectAllRecords"
```

Comment:
Just for consistency, does this work too?

```suggestion
                                x-show="canSelectAllRecords()"
```